bundles/sirix-core/src/main/java/org/sirix/access/AbstractLocalDatabase.java

```diff
@@ -65,16 +67,29 @@
    */
   protected final DatabaseConfiguration dbConfig;
 
+  private final DatabaseSessionPool sessions;
+
+  /**
+   * The resource store to open/close resource-managers.
+   */
+  protected final ResourceStore<T> resourceStore;
+
   /**
    * Constructor.
-   *
-   * @param dbConfig {@link ResourceConfiguration} reference to configure the {@link Database}
+   *  @param dbConfig {@link ResourceConfiguration} reference to configure the {@link Database}
+   * @param sessions
+   * @param resourceStore
    */
-  public AbstractLocalDatabase(final DatabaseConfiguration dbConfig) {
+  public AbstractLocalDatabase(final DatabaseConfiguration dbConfig, final DatabaseSessionPool sessions, final ResourceStore<T> resourceStore) {
+
```

Comment:
Only one minor thing, can we skip the blank line after a constructor or method declaration? Thanks a lot for all the work :+1:

Reply:
I should provide my most recent IntelliJ formatter, I think, such that we can simply auto-format :-)

Reply:
I'm using IntelliJ as well, but I installed a plugin that imports the formatter from Eclipse. Apparently it didn't work as well as I thought 🤔